### DISCUSSION TITLE
How i can load entitys with AnvilLoader?

### DISCUSSION THREAD

**Discussion by FlxwDNS** [Minestom/Minestom#1470] [Help] [answered]
Hello, I load my world with a AnvilLoader but the dont load the ArmorStands...
How i can load these?

**Comment by cody-quinn** [comment]
As of now you can't

**Reply by Smoofy19** [reply]
Is it now posibble?

**Reply by FlxwDNS** [reply]
> Is it now posibble?

Yea, you can Spawn a FakePlayer. The FakePlayer preload the chunk.

**Comment by FlxwDNS** [comment, marked as answer]
**A FakePlayer can preload the chunk.**
